src/logic/calculate.test.js

```diff
@@ -0,0 +1,9 @@
+import calculate from './calculate';
+
+describe('calculate', () => {
+  it('resets the calculator object when AC button is pressed', () => {
+    const obj = { total: '3', next: '2', operation: '+' };
+    const buttonName = 'AC';
+    expect(calculate(obj, buttonName)).toEqual({ total: null, next: null, operation: null });
+  });
+});
```

Comment:
- [ ] Hi @AlyAtalla, based on the requirements of this project, all the functionalities and scenarios of the calculate.js modules should be covered using Jest. For this specific, kindly make sure other scenarios like `buttonName === '0',  buttonName === '.', buttonName === '=', buttonName === '+/-'` are equally covered.